src/announcements/tests/test_views.py

```diff
@@ -32,21 +32,19 @@ def setUp(self):
         self.ann_pk = self.test_announcement.pk
 
     def test_all_announcement_page_status_codes_for_anonymous(self):
-        ''' If not logged in then all views should redirect to home page or admin  '''
+        ''' If not logged in then all views should redirect to home page '''
```

Comment:
this is a lie. Nothing redirects to home page, redirects to login page (ditto for other tests below)

Reply:
i think i got them all in this file